examples/tutorials/asr_inference_with_ctc_decoder_tutorial.py

```diff
@@ -68,7 +68,7 @@
 speech_url = "https://pytorch.s3.amazonaws.com/torchaudio/tutorial-assets/ctc-decoding/8461-258277-0000.wav"
 speech_file = f"{hub_dir}/speech.wav"
 
-torch.hub.download_url_to_file(speech_url, speech_file)
+torch.hub.download_url_to_file(speech_url, speech_file, progress=False)
```

Comment:
why set `progress=False`? seems beneficial for the user to get visual feedback that their code execution is making progress rather than hanging silently

Reply:
Hmm... when running, it is indeed beneficial. Meanwhile, while browsing the tutorials on browser, I found them rather distracting or unnecessarily. KenLM part adds 300 lines (though only dozens of lines are visible at a time)

https://pytorch.org/audio/main/tutorials/asr_inference_with_ctc_decoder_tutorial.html#kenlm

Reply:
ah i see didn't realize it rendered like that on the tutorial page. that being said, the removal still looks good to me